lib/assembly/consume.py

```diff
@@ -403,6 +402,7 @@ def compute(self, body):
 
         finally:
             self.remove_job_from_lists(job_data)
+            self.pmanager = ModuleManager(self.threads, self.kill_list, self.kill_list_lock, self.job_list, self.binpath)
```

Comment:
This reloads the plugin manager after job completion.  There is a mechanism in which the engine detects recursive calls, and needs to backup/restore the job environment.  (e.g. Gam-ngs uses an internal wasp expression).  

There was an issue where the plugin class was storing information and wasn't getting destroyed on job completion.